web/src/routes/admin/(resources)/grades/[id]/+page.svelte

```diff
@@ -2,45 +2,26 @@
   import AddResource from '$lib/components/ui/add-resource/add-resource.svelte';
   import { createGrade, updateGrade, type GradeRequestDto } from '$lib/sdk/fetch-client';
   import { error } from '@sveltejs/kit';
+  import { _schema as schema } from './+page';
 
   export let data;
-  let grade = data.grade;
-  let values: string[] = [grade.name];
   let descriptions: string[] = ['Name:'];
 
-  async function create(values: string[], tagIds: string[]) {
+  async function update(formData: Record<string, string | number | string[]>) {
     let gradeRequestDto: GradeRequestDto = {
-      name: values[0],
-      studentGroupIds: [],
-      tagIds,
+      name: String(formData.name),
+      studentGroupIds: formData.studentGroups as string[],
+      tagIds: formData.tags as string[],
```

Comment:
We could probably type `formData` stronger such that we don't need those casts. Not sure if it's worth it though

Reply:
I don't think typing `formData` stronger would be worth the effort.

Reply:
Most likely at this point tbh